style.css

```diff
@@ -253,7 +283,8 @@
 }
 
 .part1 {
-  display: flexbox;
+  display: flex;
```

Comment:
- [ ] You have used `display: flex` 30 times, and this falls under the category of repeating yourself, which is bad practice. I would suggest you create a CSS class similar to `d-flex` and give it `display: flex` and use it wherever you need in your HTML